d3ploy/timeseries_inst.py

```diff
@@ -239,10 +244,12 @@ def predict_supply(self, commod):
             supply = CALC_METHODS[self.calc_method](self.commodity_supply[commod],
                                                     back_steps=self.back_steps,
                                                     degree=self.degree)
+        elif self.calc_method in ['sw_seasonal']:
```

Comment:
Can we maybe just include this in the DO calc methods and have a dummy parameter `back_steps` that doesn't do anything?